ui/src/message_popup/message_popup_page.jsx

```diff
@@ -79,23 +87,54 @@ export default React.createClass({
       name: '',
       hasStarted: false,
       questionsAnswered: 0,
+      sessionLength: 20,
+      toastRevision: false,
+      responseTimes: []
     };
   },
 
   onStartPressed() {
-    const {competencyGroupValue, isSolutionMode} = this.state;
-    const questions = (isSolutionMode || competencyGroupValue === ALL_COMPETENCY_GROUPS)
-      ? _.shuffle(withStudents(allQuestions))
-      : questionsForCompetencies(competencyGroupValue);
-
+    const {competencyGroupValue} = this.state;
+    const questions = this.getQuestions();
     this.setState({
       questions,
       hasStarted: true
     });
   },
+  
+  playToast(){
```

Comment:
To me the toast UI feels like it might redundant here with the "summary" step, where the summary screen is acknowledging the response has been recorded.  What do you think?